### PR TITLE
Change the wording of multiple help tips and descriptions in the application configuration pages of the console

### PR DESCRIPTION
**Pull request by DinikaSen**
## Purpose
This PR modifies the wording of the following sections.
- the danger zone sub-heading of the Console's Application configuration's General tab
- the 'Allowed Redirect URLs' description shown when creating a new SPA
- the required filed description of 'Allowed Redirect URLs'  and its help tip in the SPA creation page.
- the help tip of 'SaaS application' checkbox in application creation's 'Advanced' tab. 
- the help tip Logo input field in General tab of application configuration.

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 1]
```diff
@@ -616,8 +616,8 @@ export const console: ConsoleNS = {
                     deleteApplication: {
                         actionTitle: "Delete",
                         header: "Delete application",
-                        subheader: "The application will be deleted permanently and the " +
-                            "clients using this application will stop working."
+                        subheader: "Once the application is deleted, it cannot be recovered and the clients " +
+                            "using this application will no longer work. Please be certain."
```
Comment (mefarazath) [inline modules/i18n/src/translations/en-US/portals/console.ts:620]: Do we need 'Please be certain.'?
Feels like it doesn't go with the rest of the wording flow.

Comment (DinikaSen) [reply modules/i18n/src/translations/en-US/portals/console.ts]: Removed 'Please be certain' phrase.

**Review by mefarazath — APPROVED** [review]

File: apps/console/src/features/applications/data/application-templates/templates/single-page-application/create-wizard-help.tsx [hunk 2]
```diff
@@ -57,11 +57,12 @@ const SinglePageApplicationCreateWizardHelp: FunctionComponent<SinglePageApplica
...
                 <p>
-                    The redirect URL determines to where the authorization code is sent, login is
-                    completed and to where the user is redirected to once the logout is completed.
+                    The redirect URL determines where the authorization code is sent to
+                    once the user is authenticated, and where the user is redirected to
+                    once the logout is completed.
                 </p>
-                <p>The client should specify the redirect_url in the authorize or logout request and
-                    &nbsp;{ config.ui.productName }&nbsp;will validate it against this configured redirect URLs.
+                <p>The redirect URLs sent by the client application along with the authentication
```
Comment (mefarazath) [inline apps/console/src/features/applications/data/application-templates/templates/single-page-application/create-wizard-help.tsx:64]: The spec usually calls these endpoints as redirect URIs.
Can we have a quick search on this and be consistent with the correct terminology?

This is how Google displays it

image

Comment (DinikaSen) [reply apps/console/src/features/applications/data/application-templates/templates/single-page-application/create-wizard-help.tsx]: `Allowed Redirect URLs` is renamed as `Authorized redirect URIs` with https://github.com/wso2/identity-apps/pull/1580

**Review by NipuniBhagya — APPROVED** [review]

**Pull request merged**